scss/bitstyles/atoms/button--ui/button--ui.stories.mdx

```diff
@@ -0,0 +1,61 @@
+import { Canvas, Meta, Story } from '@storybook/addon-docs/blocks';
+
+<Meta title="Atoms/Button/UI" />
+
+# UI Buttons
+
+Buttons that look more like native UI than the other color-branded buttons, intended to be used for actions that do not need to be highlighted, such as pagination.
+
+
+<Canvas>
+  <Story name="UI">
+    {`
+      <button type="button" class="a-button a-button--ui">
+        First
+      </button>
+    `}
+  </Story>
+</Canvas>
+
+The same buttons are used for dropdown menus
+
+<Canvas>
+  <Story name="Dropdown">
+    {`
+      <button type="button" class="a-button a-button--ui u-h6" aria-controls="element-id" aria-expanded="false">
```

Comment:
Thanks, I missed it on the first :) It’s there because most buttons need text to be slightly smaller, otherwise they become vertically large compared to other elements